modules/ROOT/pages/vpc-provisioning-concept.adoc

```diff
@@ -28,6 +30,15 @@ To enable the VPC to host your workers, it must be bound to a business group and
 
 Read the following FAQs to understand how to plan a successful VPC setup.
 
+== FAQ: How to Calculate VPC and VPN Licensing Requirements
+
+To connect a CloudHub VPC through IPSec VPN, you must have the required number of licenses for your deployment scenario. 
+
+The Anypoint Platform base subscription includes two VPC licenses, which entitles your Anypoint Platform organization to two VPCs. VPN licenses are bundled with the VPC license, so each VPC license also entitles the organisation to one VPN gateway with a connection to one public IP address in a remote location. Therefore, one VPC License = 1 VPC entitlement + 1 VPN entitlement.
```

Comment:
Do we want to call out that the VPN entitlement doesn't need to be used 1:1 with the VPC entitlement? E.g. if I have two VPCs, I can have two VPNs in a VPC, and none in the other VPC